src/test/variable/VRLadle.t.sol

```diff
@@ -154,18 +154,6 @@ contract VaultTests is VaultBuiltState {
         ladle.give(vaultId, admin);
     }
 
-    function testOnlyOwnerCouldMove() public {
```

Comment:
Why remove so many tests from this file? Is it because they're already tested elsewhere?

Reply:
The tests related to `stir` were removed.